src/functions-and-arrays.js

```diff
@@ -52,14 +141,39 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(inputArray) {
+
+  if (inputArray.length > 0) {
+    
+    let uniqueArray = [];
+  
+    for (i = 0; i < inputArray.length; i++) {
+      if (uniqueArray.includes(inputArray[i])) {
+        continue;
+      } else {
+        uniqueArray.push(inputArray[i]);
+      }
```

Comment:
Here instead of using both `if` and `else`, it is possible to negate a condition by adding exclamation point before the condition. for example:

` if (!uniqueArray.includes(inputArray[i])) {
        uniqueArray.push(inputArray[i]);
   }`

This says, if array does not include the element -> push it into array